gensim/models/ldamodel.py

```diff
@@ -192,10 +195,11 @@ class LdaModel(interfaces.TransformationABC, basemodel.BaseTopicModel):
     """
     def __init__(self, corpus=None, num_topics=100, id2word=None,
                  distributed=False, chunksize=2000, passes=1, update_every=1,
-                 alpha='symmetric', eta=None, decay=0.5, offset=1.0,
-                 eval_every=10, iterations=50, gamma_threshold=0.001,
-                 minimum_probability=0.01, random_state=None, ns_conf={},
-                 minimum_phi_value=0.01, per_word_topics=False):
+                 alpha='symmetric', eta=None, decay=0.5, offset=1.0, eval_every=10,
+                 iterations=50, gamma_threshold=0.001, minimum_probability=0.01,
+                 random_state=None, ns_conf={}, minimum_phi_value=0.01,
+                 per_word_topics=False, viz=False, env=None, distance="kulback_leibler",
```

Comment:
Validation about every arg's object type? Or for the valid values of 'diff_distance' and 'coherence' (though, for which there is a test already in their respective functions of [Diff](https://github.com/RaRe-Technologies/gensim/blob/develop/gensim/models/ldamodel.py#L1002) and [CoherenceModel](https://github.com/RaRe-Technologies/gensim/blob/develop/gensim/models/coherencemodel.py#L194))?